coffeecard/CoffeeCard.Models/Entities/PosPurchase.cs

```diff
@@ -0,0 +1,13 @@
+namespace CoffeeCard.Models.Entities
+{
+    /// Represents a purchase issued by a barista at the point of sale in the cafe
+    public class PosPurhase
+    {
+        /// Primary key
+        public int Id { get; set; }
+        /// The purchase that was issued by the barista
+        public Purchase Purchase { get; set; }
```

Comment:
I think we would need a property PurchaseId which we can query on to avoid having to eagerly load the Purchase property if we need to work with it